hw3/01-characters.js

```diff
@@ -1,4 +1,75 @@
 // url for the Thrones API
 const url = 'https://thronesapi.com/api/v2/Characters';
+const sectionToAddTo = document.querySelector('section');
+sectionToAddTo.classList.add('container-fluid', 'row');
 
-fetch(url);
+// fetch the data from the API
+fetch(url)
+  .then((data) => {
+    return data.json();
+  }) // return the json of the data
+  .then((characters) => {
+    displayCharacters(characters, sectionToAddTo);
+  }) // call fn to display it (append to the given element)
+  .catch((error) => {
+    console.log(error);
+  }) // catch any errors and throw them at the console
+  .finally(() => {
+    console.log('API call completed');
```

Comment:
This console output could be cleaned up.

Reply:
Fair. Removed